epub33/core/index.html

```diff
@@ -1575,14 +1575,13 @@ <h6>The <code>identifier</code> Element</h6>
 									</dd>
 								</dl>
 
-								<p>The <a>Author</a> MUST provide a primary identifier that is unique to one and only
-									one <a>EPUB Publication</a> in the <a href="#sec-opf-dcidentifier"
-											><code>dc:identifier</code> element</a>. This identifier MUST be referenced
-									as the Unique Identifier in the <a href="#elemdef-opf-package"><code>package</code>
-										element's</a>
+								<p>The <a>Author</a> MUST provide an identifier that is unique to one and only one
+										<a>EPUB Publication</a> &#8212; its <a>Unique Identifier</a> &#8212; in an
+										<code>identifier</code> element. This <code>identifier</code> element MUST
+									include an <code>id</code> attribute whose value is referenced from the <a
```

Comment:
Shouldn't it be "contain" rather than "include"? I would not want the content of the element to be something like "https://identify.er obladi oblada" :-)

Reply:
Don't think so, but maybe we're not reading it the same way. An `id` attribute only ever has a single value, and the phrasing is "must include an id attribute whose value is referenced from". I don't see a suggestion there that the id can have more than one value or that only part of the value is matched.

"Contain" is typically used to refer to the value / child content space of the attribute or element it refers to.

Looking around at some other specs for guidance, I'm not finding a consistent phrasing for referring to attributes of elements. The XML specification likes to use "attach", HTML prefers "specify", SVG uses "include" and MathML seems to mix "include" and "specify".

Of the options, I'd probably go with "specify" if we want to harmonize on something other than including. But I'm not too concerned either way.

Reply:
Let us go with specify. Just to be in line with HTML :-)